### ISSUE TITLE
Issue with accessing the repo to install the package

### ISSUE THREAD

**Issue by sfallahpour** [markvanderloo/simputation#38]
@markvanderloo when i use this command to install the package from github

```
library(devtools)
install_github("markvanderloo/simputation")
```

i get the following error:

```
Error: Failed to install 'unknown package' from GitHub:
  HTTP error 404.
  No commit found for the ref MultipleTimeSeries

  Did you spell the repo owner (`markvanderloo`) and repo name (`simputation`) correctly?
  - If spelling is correct, check that you have the required permissions to access the repo.
```

what is alternative to use the updated version with all the fixes. The library(simputation) still doesnt include the current fixes so  i thought to install it directly from github.

**Comment by markvanderloo** [comment]
Hi there, the installation instructions for the development version are on the front page of this repo. At the moment the CRAN submission cue is closed so I need to wait a week before submitting the last version. If you're on Windows you'll need to instal [Rtools](https://cran.r-project.org/bin/windows/Rtools/)

**Comment by sfallahpour** [comment]
Ok great Would you please let me know once you submit it to the Cran. Really appreciate itSent from my iPhoneOn Dec 29, 2023, at 5:05 AM, Mark van der Loo ***@***.***> wrote:﻿
Closed #38 as completed.
